packages/headless/src/features/facets/facet-set/facet-set-slice.ts

```diff
@@ -281,3 +317,18 @@ function restoreFacetValueToIdleState(
 ): FacetValueRequest {
   return {...facetValue, state: 'idle'};
 }
+
+function deleteOldAutomaticFacets(
```

Comment:
what does old means here? i.e. when does an automatic facets become old?

Reply:
For the upper comment about the `static` state. I decided to put the logic for this inside of the `facetRequestSelector` in the `facet-set-selector.ts` instead. Thank you for the comment.

Reply:
I could replace the name for `removeUnusedAutomaticFacets` or `cleanupAutomaticFacets` what do you prefer ?